### DISCUSSION TITLE
Ability to change stack direction

### DISCUSSION THREAD

**Discussion by MightyHelper** [ppy/osu#13090] [Ideas]
**Describe the new feature:** 
When two circles are close in time and in the same place, osu!stable stacks them.
I suggest that you can change the _angle_ at which the circles are displaced.
**To be decided:** 
- Changing this setting affects previous occurrences in the map?
- Also, change stack distance?

**Comment by iSimpFor** [comment]
i suggest adding them to the inherited points and uninherited points so that changing the angle in that will only affect that section

(stack distance also sounds good)
